content/influxdb/v2.1/notebooks/downsample.md

```diff
@@ -0,0 +1,113 @@
+[create a new bucket](/influxdb/v2.1/organizations/buckets/create-bucket/).
+{{% /note %}}
+
+1. [Create a new notebook](/influxdb/v2.1/notebooks/create-notebook/).
+2.  In the **Build a Query** cell:
+
+    1.  In the **FROM** column under **{{% caps %}}Sample{{% /caps %}}**,
+        select **Coinbase bitcoin price**.
+    2.  In the next **FILTER** column, select **_measurement** from the drop-down list
+        and select the **coindesk** measurement in the list of measurements.
+    3.  In the next **FILTER** column, select **_field** from the drop-down list,
+        and select the **price** field from the list of fields.
+    4.  In the next **FILTER** column, select **code** from the drop-down list,
+        and select a currency code from the list.
+
+3.  Click {{% icon "notebook-add-cell" %}} after your **Build a Query** cell to 
+    add a new cell and select **{{% caps %}}Flux Script{{% /caps %}}**.
+
+4.  In the Flux script cell:
+    
+    1.  Use `__PREVIOUS_RESULT__` to load the output of the previous notebook
+        cell into the Flux script.
+    3.  Use [`aggregateWindow()`](/{{< latest "flux" >}}/stdlib/universe/aggregateWindow/)
+        to window data into one hour intervals and return the average of each hour.
+        Specify the following parameters:
+
+        - **every**: Window interval _(should be less than or equal to the selected query time range)_.
+        - **fn**: [Aggregate]({{< latest "flux" >}}/function-types/#selectors)
+          or [selector function]({{< latest "flux" >}}/function-types/#selectors)
+          to apply to each window.
+
+    4.  {{% cloud-only %}}
+        
+        Use [`to()`](/{{< latest "flux">}}/stdlib/influxdata/influxdb/to/)
+        to write the downsampled data back to an InfluxDB bucket.
+
+        {{% /cloud-only %}}
+    
+    {{% oss-only %}}
+
+    ```js
+    __PREVIOUS_RESULT__
+        |> aggregateWindow(every: 1h, fn: mean)
+    ```
+    {{% /oss-only %}}
+
+    {{% cloud-only %}}
+
+    ```js
+    __PREVIOUS_RESULT__
+        |> aggregateWindow(every: 1h, fn: mean)
+        |> to(bucket: "example-bucket")
+    ```
+    {{% /cloud-only %}}
+
+4.  {{% oss-only %}}
+    
+    Click {{% icon "notebook-add-cell" %}} after your **Flux Script** cell to 
+    add a new cell and select **{{% caps %}}Output to Bucket{{% /caps %}}**.
+    Select a bucket from the **{{% icon "bucket" %}} Choose a bucket**
+    drop-down list. 
+
+    {{% /oss-only %}}
+    
+5.  _(Optional)_ Click {{% icon "notebook-add-cell" %}} and select **Note** to
+    add a cell containing notes about what this notebook does. For example, the
+    cell might say, "This notebook downsamples Coinbase bitcoin prices into hourly averages."
+6.  {{% oss-only %}}
+
+    Click **Preview** in the upper left to verify that your notebook runs and previews the output.
+    
+    {{% /oss-only %}}
+6.  Click **Run** to run the notebook and write the downsampled data to your bucket.
+
+## Continuously run a notebook
```

Comment:
Thoughts on _Continuously write data using a notebook_...possibly more explicit, but maybe not necessary?

Reply:
Notebooks don't necessarily write data. This particular example does. I don't know that it's necessary to say that.